### PR TITLE
1019/inform users

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1019 .

### CODE REVIEW

File: Document/0x04i-Testing-user-interaction.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Testing User Education
+A lot has happened lately in terms of responsibilities that developers have to educate users on what they need to know.
+This has shifted especially with the introduction of the [General Data Protection Regulation (GDPR)](https://gdpr-info.eu/ "GDPR") in Europe. Ever since then, it is best to educate users on what is happening with their private data and why.
+Next, it is a good practice to inform the user about how he can best use the application to ensure a secure processing of his information.
+Both items will be dealt with here. Please not that this is the MSTG project and not a legal handbook. Therefore, we will not cover the GDPR and other possibly relevant laws here.
+
+
+### Informing users on their private information
+When you need personal information from a user for your business process, then the user needs to be informed what you do with the data and why you need it. If there is a third party doing the actual processing of the data, then the user should be informed about that. Lastly, there are three processes you need to support:
+- The right to be forgotten: A user needs to know how - and be be able to request the deletion of his data.
```
Comment (TheDauntless) [inline Document/0x04i-Testing-user-interaction.md]: Kind of a weird way of saying it, and I can't seem to fix it :p How about just the following? Or do you want to make a distinct difference between providing functionality, and informing the user on where that functionality is?

> The right to be forgotten: A user needs to be able to request the deletion of his data.

File: Document/0x04i-Testing-user-interaction.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- The right to correct the data: the user should know - and should be able to correct his personal information at any time.
```
Comment (TheDauntless) [inline Document/0x04i-Testing-user-interaction.md]: > The right to correct data: The user should be able to correct his personal information at any time.

File: Document/0x04i-Testing-user-interaction.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- The right to access by the data subject: the user always has the right to request all the data that you have on him/her.
```
Comment (TheDauntless) [inline Document/0x04i-Testing-user-interaction.md]: > The right to access user data: The user always has the right to request all the data that you have on him.

Additionally, while I personally write reports with him/her, I think this is the first time we do it in the MSTG. So unless we define a style guide, let's keep this as "him" too.

**Comment by commjoen** [comment]
Merged in order to take all spelling of the 3 PRs together (#1020, #1021, #1022)

**Pull request merged**